tests/common.bash

```diff
@@ -336,3 +333,83 @@ function check_containerd_config_for_kata() {
 		overwrite_containerd_config
 	fi
 }
+
+function ensure_yq() {
+    : "${GOPATH:=${GITHUB_WORKSPACE:-$HOME/go}}"
```

Comment:
mm why we have the `:` here?

Reply:
That's, unfortunately, a GHA convention. :-/